pkg/data/lease.go

```diff
@@ -4,20 +4,27 @@ import (
 	"fmt"
 
 	"github.com/Optum/dce/pkg/errors"
-	"github.com/Optum/dce/pkg/model"
-
+	"github.com/Optum/dce/pkg/lease"
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/awserr"
 	"github.com/aws/aws-sdk-go/service/dynamodb"
 	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbiface"
 	"github.com/aws/aws-sdk-go/service/dynamodb/expression"
 )
 
-// WriteLease the Lease record in DynamoDB
+// Lease - Data Layer Struct
+type Lease struct {
```

Comment:
Need to have the Lease stuff separate.  I think we are on a path to have a generic Data service but need to figure out how to handle some of the inconsistencies between the tables first.